koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -536,41 +451,192 @@ def check_whitelist(self, name, target_info):
+                "type": "string",
+                "description": "Source URI."
+            },
+            {
+                "type": "string",
+                "description": "Build target."
+            },
+            {
+                "type": ["object"],
+                "properties": {
+                    "scratch": {
+                        "type": "boolean",
+                        "description": "Perform a scratch build?"
+                    },
+                    "isolated": {
+                        "type": "boolean",
+                        "description": "Perform an isolated build?"
+                    },
+                    "yum_repourls": {
+                        "type": ["array", "null"],
+                        "items": {
+                            "type": "string"
+                        },
+                        "description": "URLs of yum repo files."
+                    },
+                    "arch_override": {
+                        "type": ["string", "null"],
+                        "description": "Limit build to specific arches"
+                    },
+                    "git_branch": {
+                        "type": ["string", "null"],
+                        "description": "Git branch to build from."
+                    },
+                    "push_url": {
+                        "type": ["string", "null"]
+                    },
+                    "koji_parent_build": {
+                        "type": ["string", "null"],
+                        "description": "Overwrite parent image with image from koji build."
+                    },
+                    "release": {
+                        "type": ["string", "null"],
+                        "description": "Set release value."
+                    },
+                    "flatpak": {
+                        "type": "boolean",
+                        "description": "Build a flatpak instead of a container?"
+                    },
+                    "compose_ids": {
+                        "type": ["array", "null"],
+                        "items": {
+                            "type": "integer"
+                        },
+                        "description": "ODCS composes used."
+                    },
+                    "signing_intent": {
+                        "type": ["string", "null"],
+                        "description": "Signing intent of the ODCS composes."
+                    },
+                    "skip_build": {
+                        "type": "boolean",
+                        "description": "Skip build, just update buildconfig for autorebuild "
+                                       "and don't start build"
+                    },
+                    "triggered_after_koji_task": {
+                        "type": "integer",
+                        "description": "Koji task for which autorebuild runs"
+                    }
+                },
+                "additionalProperties": False
```

Comment:
This is a new restriction. Are we happy adding it? Is there any likelihood of tasks being submitting with extra parameters not listed here?

Reply:
all parameters which we are passing to osbs are here, so I think it is fine